DER/DerivativesContracts/RightsAndWarrants.rdf

```diff
@@ -181,6 +341,34 @@
 		<skos:definition xml:lang="en">warrant that is listed on a securities exchange</skos:definition>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-der-drc-raw;IndexWarrant">
+		<rdfs:subClassOf rdf:resource="&fibo-der-drc-raw;Warrant"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-der-drc-bsc;hasUnderlier"/>
+				<owl:someValuesFrom>
+					<owl:Restriction>
+						<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;hasIdentity"/>
```

Comment:
what class will the underlier be that it in turn hasIdentity? hasIdentity itself seems wrong for this.